lib/IDE/CodeCompletion.cpp

```diff
@@ -2268,10 +2244,49 @@ class CompletionLookup final : public swift::VisibleDeclConsumer {
         addTypeAnnotation(Builder, AFT->getResult());
     };
 
-    if (hasInterestingDefaultValues(AFD))
-      addPattern(/*includeDefaultArgs*/ false);
-    addPattern();
+    if (!AFD || !AFD->hasInterfaceType() ||
+        !AFD->getInterfaceType()->is<AnyFunctionType>()) {
+      // Probably, calling closure type expression.
+      foundFunction(AFT);
+      addPattern();
+      return;
+    } else {
+      // Calling function or method.
+      foundFunction(AFD);
+
+      // FIXME: Hack because we don't know we are calling instance
```

Comment:
It should be easy to know this -- what is the underlying issue?

Reply:
There's no issue. I will fix this when I have time.
I think it need to look into the original expression, right?